cmd/thor/solo/solo.go

```diff
@@ -196,3 +204,53 @@ func (s *Solo) packing(pendingTxs tx.Transactions, onDemand bool) error {
 
 	return nil
 }
+
+// initSolo adds transactions to the first block to set the chain state
+func (s *Solo) initSolo() error {
+	gasPriceTx, err := s.makeGasPriceTx()
+	if err != nil {
+		return err
+	}
+
+	if err := s.txPool.Add(gasPriceTx); err != nil {
+		return err
+	}
+
+	return s.packing(tx.Transactions{gasPriceTx}, false)
```

Comment:
This line is not necessary, should let the packing loop decide when to pack.

Reply:
Do you know how we can force it to be in the first block that gets packed? When I don't call `s.packing(..)` it always excludes it from the first block to get packed. Ideally it would happen on the first block for CI /CD

<img width="1388" alt="image" src="https://github.com/vechain/thor/assets/107671032/cf0151e8-414c-4235-bf22-ba67105c5e0e">

Reply:
I see, it was caused by some internal logic of the tx-pool, blockchain is not 'synced' when the genesis block time was a long time ago, it needs an up-to-date block to make it ready to produce pending tx. I was trying to make block time follow the interval(for none-on-demand mode), but it wasn't ideal for this case, if you wish you make a on-demand block for that.